src/test/resources/pages/instructorFeedbackQuestionVisibilityOptions.html

```diff
@@ -1011,7 +1011,7 @@ <h5 class="col-sm-2">
                   <option selected="" value="TEAMS">
                     Other teams in the course
                   </option>
-                  <option style="display: none;" value="OWN_TEAM">
+                  <option style="display: block;" value="OWN_TEAM">
```

Comment:
Looks like this is unintended? This question has feedback path from "Teams in this course > ..." which should mean "Giver's team" cannot be selected.